templates/bootstrap/views/index.php

```diff
@@ -1,38 +1,41 @@
 <?php
 
-use yii\apidoc\models\ClassDoc;
-use yii\apidoc\models\InterfaceDoc;
-use yii\apidoc\models\TraitDoc;
+use yii\apidoc\helpers\ApiMarkdown;
 
-/* @var $types ClassDoc[]|InterfaceDoc[]|TraitDoc[] */
-/* @var $this yii\web\View */
-/* @var $renderer \yii\apidoc\templates\html\ApiRenderer */
+/**
+ * @var yii\apidoc\models\ClassDoc[]|yii\apidoc\models\InterfaceDoc[]|yii\apidoc\models\TraitDoc[] $types
```

Comment:
I'd leave explicit import of classes used in PHPDoc. Especially with unions like this it's more readable.

Reply:
@arogachev PHPDoc created not to be human readable